api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -336,10 +337,17 @@ func (c *AerospikeCluster) validateRackUpdate(
 				}
 
 				if len(oldRack.AerospikeConfig.Value) != 0 || len(newRack.AerospikeConfig.Value) != 0 {
+					var statusRack Rack
```

Comment:
statusRack will become the last rack from c.Status.RackConfig.Racks.

Reply:
fixed